colors/classic.yaml

```diff
@@ -4,20 +4,27 @@ name: Monokai Pro (Classic)
 colors:
   # primary colors
   red: "#f92672"
+  errorRed: "#990909"
   orange: "#fd971f"
   yellow: "#a9dc76"
+  darkYellow: "#5aa90e"
   green: "#a6e22e"
+  darkGreen: "#117e00"
   blue: "#66d9ef"
+  darkBlue: "#0050bb"
   purple: "#ab9df2"
   # base colors, sorted from darkest to lightest
-  base0: "#161613"
-  base1: "#1d1e19"
+  base0: "#13130f"
+  base1: "#1e1e1a"
+  base1.5: "#20211c"
```

Comment:
You need to change the name, `.` in the key is not supported. You might have to use `_` and replace in theme template as well.

You can refer to the built theme files.

Reply:
Can you drop a mail to me at the email mentioned on my GitHub profile? Your efforts looks more promising than the original dev of this project. 😁